migrations/000017_vacuum.down.sql

```diff
@@ -0,0 +1,28 @@
+-- Copyright 2020 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+ALTER TABLE APIConfig SET (autovacuum_enabled = false);
+ALTER TABLE ExportBatch SET (autovacuum_enabled = false);
+ALTER TABLE ExportConfig SET (autovacuum_enabled = false);
+ALTER TABLE ExportFile SET (autovacuum_enabled = false);
+ALTER TABLE Exposure SET (autovacuum_enabled = false);
```

Comment:
I think we need to tune this param lower

https://www.postgresql.org/docs/8.4/runtime-config-client.html#GUC-VACUUM-FREEZE-TABLE-AGE

the default is to wait for up to 150 million transactions - which is many many days

Reply:
perhaps others. - @jba - do you have any thoughts?

Reply:
We seem to use whatever defaults GCP gives us. The only things we change are
```
    {
      name  = "autovacuum_vacuum_cost_limit"
      value = 10000
    },
    {
      name  = "vacuum_cost_limit"
      value = 10000
    }
```

Reply:
Vacuuming is _very_ expensive and acquires a full table lock. Do we want to do it more frequently?